core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/ColumnPageEncoder.java

```diff
@@ -78,6 +78,13 @@ public DataType getTargetDataType(ColumnPage inputPage) {
     }
   }
 
+  public Encoding getTypeOfEncoding() {
+    if (CarbonUtil.isEncodedWithMeta(getEncodingList())) {
+      return getEncodingList().get(0);
```

Comment:
1. `getEncodingList()` is called 2 times. Hold its value in one variable and use at both the places
2. Change the method name from `getTypeOfEncoding()` to `getEncodingType()`